src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -198,3 +198,15 @@ def test_writeonly_attribute(device_info):
             assert e.code == -1074135027 #Error : Attribute is read-only.
 
 
+def test_invalid_value_attribute(device_info):
+    with nidmm.Session(device_info['name']) as session:
+        try:
+            session.settle_time = 5
```

Comment:
If this doesn't raise, then your test will pass. This would be a false positive.
You need to assert False after the line that is supposed to raise.